client/src/views/Profile.jsx

```diff
@@ -0,0 +1,351 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Container from "react-bootstrap/Container";
+import Profilephoto from "../assets/ImageLarge.png";
+import { useHistory } from "react-router-dom";
+import axios from "axios";
+import { FaCheckCircle, FaBullseye, FaAngleRight } from "react-icons/fa";
+const Profile = ({ auth }) => {
+  const history = useHistory();
+  const ClickOngoing = (e) => {
+    e.preventDefault();
+    history.push("/goals");
+  };
+
+  const DeleteHandler = (e) => {
+    e.preventDefault();
+    axios
+      .delete("/user", {
+        withCredentials: true,
+      })
+      .then((response) => {
+        alert("Account deleted");
+        history.push("/onboarding");
+      })
+      .catch((err) => {
+        alert("Cannot delete account");
+      });
+  };
+
+  return (
+    <Wrapper>
+      <CustomContainer>
+        <Title>Profile</Title>
+        <InformationWrapper>
+          <BlockDiv>
+            <ProfileImage src={Profilephoto} />
+            <Name>{auth.username}</Name>
+          </BlockDiv>
+          <FlexDiv>
+            <Line />
+            <BlockDiv>
+              <Joined>Joined</Joined>
+              <FlexDiv>
+                <Times>
+                  {Math.floor(
+                    (Date.now() - Date.parse(auth.joinDate)) /
+                      (1000 * 3600 * 24)
+                  )}
+                </Times>
+                <DaysAgo> days ago</DaysAgo>
+              </FlexDiv>
+            </BlockDiv>
+          </FlexDiv>
+        </InformationWrapper>
+
```

Comment:
I think the goals and completed goals have to directly show here instead of routing, as we also want people to click in other's profile from leaderboard